### PR TITLE
Enable licensing and fault domains for 1.11

### PR DESCRIPTION
**Pull request by amitaekbote**

### CODE REVIEW

**Comment by orsenthil** [comment]
@amitaekbote  - minor comment, plus - please share how did you test this change? 

Like

**testing done**
```
$ curl -O dcos_generate_config.ee.sh
$ ./configure 
$ make
$ make postflight
$ UI came up successfully and no failures.
```

**Comment by amitaekbote** [comment]
Tested 1.11 open and enterprise with the pr

File: Vagrantfile [hunk 1]
```diff
@@ -74,6 +75,7 @@ class UserConfig
     c.version              = ENV.fetch(env_var('version'), '')
     c.generate_config_path = ENV.fetch(env_var('generate_config_path'), '')
     c.install_method       = ENV.fetch(env_var('install_method'), 'ssh_pull')
+    c.license_key_contents = ENV.fetch(env_var('license_key_contents'), '')
```
Comment (orsenthil) [inline Vagrantfile]: @amitaekbote  - I tried this.


```
$ echo $LICENSE_KEY_CONTENTS
license.txt
$ ls -1 license.txt
license.txt
==> boot: Importing Private SSH Key: ~/dcos/genconf/ssh_key
==> boot: [sudo]$ cp /vagrant/.vagrant/dcos/private_key_vagrant ~/dcos/genconf/ssh_key
==> boot: Generating DC/OS Installer Files: ~/dcos/genconf/serve/
==> boot: [sudo]$ cd ~/dcos && bash ~/dcos/dcos_generate_config.sh --genconf && cp -rpv ~/dcos/genconf/serve/* /var/tmp/dcos/ && echo ok > /var/tmp/dcos/ready
==> boot:       ====> EXECUTING CONFIGURATION GENERATION
==> boot:       Generating configuration files...
==> boot:       license_key_contents: ERROR: No license key provided
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

cd ~/dcos && bash ~/dcos/dcos_generate_config.sh --genconf && cp -rpv ~/dcos/genconf/serve/* /var/tmp/dcos/ && echo ok > /var/tmp/dcos/ready

Stdout from the command:



Stderr from the command:

====> EXECUTING CONFIGURATION GENERATION
Generating configuration files...
license_key_contents: ERROR: No license key provided
```

Comment (orsenthil) [reply Vagrantfile]: I also tried with having the entire license key copied into the variable and that still have the same issue.

**Pull request merged**